### PR TITLE
feat(ROI Calculator): Farms displays both LP and CAKE rewards

### PR DESCRIPTION
**Pull request by woodywu810**
Farms Add LP rewards info.

### CODE REVIEW

**Review by chefilip — APPROVED** [review]

File: src/config/localization/translations.json [hunk 1]
```diff
@@ -1768,5 +1768,6 @@
   "to try.": "to try.",
   "%percentageVested%% of the purchased token will get vested and released linearly over a period of time. %percentageTgeRelease%% of the purchased token will be released immediately and available for claiming when IFO ends.": "%percentageVested%% of the purchased token will get vested and released linearly over a period of time. %percentageTgeRelease%% of the purchased token will be released immediately and available for claiming when IFO ends.",
   "The vested tokens will be released linearly over a period of %weeks% weeks.": "The vested tokens will be released linearly over a period of %weeks% weeks.",
-  "Select a Network": "Select a Network"
+  "Select a Network": "Select a Network",
+  "LP rewards only": "LP rewards only"
```
Comment (chefilip) [inline src/config/localization/translations.json:1772]: I think it would be more precise by using `LP Rewards APR`

Comment (woodywu810) [reply src/config/localization/translations.json]: Corrected.

File: src/components/RoiCalculatorModal/RoiCalculatorFooter.tsx [hunk 2]
```diff
@@ -103,6 +103,16 @@ const RoiCalculatorFooter: React.FC<RoiCalculatorFooterProps> = ({
             <Text small textAlign="right">
               {(apy ?? apr).toFixed(2)}%
             </Text>
+            {isFarm && (
+              <>
+                <Text color="textSubtle" small>
+                  *{t('LP Rewards APR')}
+                </Text>
+                <Text small textAlign="right">
+                  {(Number(displayApr) - apr).toFixed(2)}%
```
Comment (ChefMomota) [inline src/components/RoiCalculatorModal/RoiCalculatorFooter.tsx:112]: Prefer
```
const lpReward = (Number(displayApr) - apr).toFixed(2)

<Text small textAlign="right">
 {`${lpReward}%`}
</Text>

or

const lpReward = useMemo(() => {
   const reward = (Number(displayApr) - apr).toFixed(2)
   return `${reward}%`
},[apr, displayApr])

<Text small textAlign="right">
 {lpReward}
</Text>

```

Comment (woodywu810) [reply src/components/RoiCalculatorModal/RoiCalculatorFooter.tsx]: Corrected.

**Review by ChefMomota — DISMISSED** [review]

**Review by chefilip — APPROVED** [review]

**Review by chefilip — APPROVED** [review]

**Pull request merged**